bindings/generator.go

```diff
@@ -0,0 +1,26 @@
+package bindings
+
+import (
+	"fmt"
+	"log"
+
+	js "github.com/onflow/flixkit-go/bindings/js"
+	"github.com/onflow/flixkit-go/common"
+)
+
+func Generate(lang string, flix *common.FlowInteractionTemplate, templateLocation string) (string, error) {
```

Comment:
If the goal is to run this from Flow CLI I don't think this makes sense to be here. It should instead pull the file and get the command using the CLI and then call the fundings it needs to which occur inside the case statements. That's what happens here in the Flow CLI for running FLIX: https://github.com/onflow/flow-cli/blob/master/internal/super/flix.go#L104

Then you can also use Cobra in the CLI which will handle all this better and remove the need for the cmd folder.